checkov/cloudformation/cfn_utils.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 
+from copy import deepcopy
```

Comment:
we have the `pickle_deepcopy` func for performance wise :)

Reply:
Great I'll use it, Thanks!